vividus-plugin-parquet/build.gradle

```diff
@@ -0,0 +1,16 @@
+project.description = 'Vividus plugin for Parquet'
+ext.sonarProjectName = 'Vividus Plugin - Parquet'
+
+dependencies {
+    api project(':vividus-bdd-engine')
+    implementation project(':vividus-util')
+    implementation project(':vividus-plugin-csv')
+
+    implementation(group: 'org.apache.parquet', name: 'parquet-avro', version: '1.10.1')
+    implementation(group: 'org.apache.hadoop', name: 'hadoop-common', version: '3.2.0')
+
+    testImplementation platform(group: 'org.junit', name: 'junit-bom', version: versions.junit)
+    testImplementation(group: 'org.junit.jupiter', name: 'junit-jupiter')
+    testImplementation(group: 'org.hamcrest', name: 'hamcrest', version: versions.hamcrest)
+    testImplementation(group: 'org.apache.hadoop', name: 'hadoop-mapreduce-client-core', version: '3.2.0')
```

Comment:
does it make sense to extract variable like `hadoopVersion`? example: https://github.com/vividus-framework/vividus/blob/master/vividus-plugin-web-app/build.gradle#L5